dropwizard-testing/src/test/java/io/dropwizard/testing/junit/DropwizardServiceRuleResetConfigOverrideTest.java

```diff
@@ -0,0 +1,35 @@
+package io.dropwizard.testing.junit;
+
+import org.junit.FixMethodOrder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import static io.dropwizard.testing.junit.ConfigOverride.config;
+import static io.dropwizard.testing.junit.DropwizardAppRuleTest.resourceFilePath;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
I've changed this test to just call the Rule methods before() and after() directly. No need to test JUnit itself. ;)